### DISCUSSION TITLE
Compile Quartz.Web

### DISCUSSION THREAD

**Discussion by alessioGit83** [quartznet/quartznet#1597] [Q&A]
Hello Guys,
I download the head version, compile all the proejcts with the .cmd files, all fine.
Now I'm trying to compile the Quartz.Web but after download all packege with npm and jspm still have around 200 errors like :
- TS2307	(TS) Cannot find module 'MODULE_NAME' or its corresponding type declarations.
- TS2314	(TS) Generic type 'Promise<T, R>' requires 2 type argument(s).

Can someone help me with this? there is a guide or something to follow in order to compile properly this project ?

Thanks
Regards
Ale

**Comment by lahma** [comment]
Honestly it's been a dormant project for now, just a POC that was left to gather dust. For good options I would suggest you check [this discussion](https://github.com/quartznet/quartznet/discussions/1079) which contains links to working, battle-tested 3rd party implementations.

Quartz will need good HTTP API as priority before even considering building something using remote communication, and ideally we could use already existing implementations.
